_pages/product.md

```diff
@@ -29,9 +29,9 @@ login.gov offers the public a single, secure, and private access to online feder
 
 ## login.gov's offerings
 
-login.gov offers secure two-factor authentication with widely available as well as unphishable methods. This service provides users with secure access to digital services at over 17 different government agencies with a single set of credentials. For you, this means leveraging the best security defined by NIST 800-63 Identity Assurance Level 1 and Authentication Assurance Level 2, and FedRAMP Moderate while complying with 21st Century Idea Act in a easy-to-use experience.
+login.gov offers secure two-factor authentication with widely available as well as unphishable methods. This service provides users with secure access to digital services at over 17 different government agencies with a single set of credentials. For you, this means leveraging the best security defined by NIST 800-63 Identity Assurance Level 1 (IAL1) and Authentication Assurance Level 2 (AAL2), and FedRAMP Moderate while complying with 21st Century Idea Act in an easy-to-use experience.
```

Comment:
Maybe not here, but this reminds me we ought to do a pass for capitalization on this site / page ("login.gov" vs. "Login.gov").

Reply:
Totally agree! But I think we're sunsetting this entire site for the partners site migration to `identity-site` too